src/apis/countryFlag.ts

```diff
@@ -0,0 +1,33 @@
+export interface CountryFlagItem {
+  content_ty: null | string;
+  country_eng_nm: string;
+  country_iso_alp2: string;
+  country_nm: string;
+  download_url: string;
+  origin_file_nm: string | null;
+}
+
+interface CountryFlagResponse {
+  response: {
+    body: {
+      dataType: string;
+      items: {
+        item: CountryFlagItem[];
+      };
+    };
+  };
+}
+// 한번의 불러올 데이터 양 (제공하는 최대 나라 수)
+const NUM_ROWS = 220 as const;
```

Comment:
서버에서 지정한 변수명 그대로 쓰기보다, 가독성 좋게 수정하면 좋을 것 같아요! 아래는 예시이니 현재 상황에 어울리는 변수명으로 수정하시면 좋을 것 같아요~

```suggestion
const SOME_MEANINGFUL_COUNT = 220 as const;
```